packages/utils/__snapshots__/ast-utils.test.js.snap

```diff
@@ -12,29 +12,29 @@ exports[`utils addProperty add entry property using add 1`] = `
                   }],
 
                   man: () => duper,
-                  objects: are,
+                  objects: 'are',
```

Comment:
See what I mentioned about the string stuff. This is invalid syntax for the ast construction.